auth-server/test/resolvers/mutation/addressLinkConfirm.spec.ts

```diff
@@ -0,0 +1,101 @@
+import { expect } from 'chai';
+import { AuthenticationError, ForbiddenError } from 'apollo-server';
+import 'mocha';
+
+import User from '../../../src/model/User';
+import Address from '../../../src/model/Address';
+import addressLinkStart from '../../../src/resolvers/mutation/addressLinkStart';
+import addressLinkConfirm from '../../../src/resolvers/mutation/addressLinkConfirm';
+import signup from '../../../src/resolvers/mutation/signup';
+import { Context } from '../../../src/types';
+import messages from '../../../src/utils/messages';
+
+describe('addressLinkConfirm mutation', () => {
+	let signupResult;
+	const fakectx: Context = {
+		req: {
+			headers: {}
+		},
+		res: {
+			cookie: () => {}
+		}
+	} as any;
+	const email = 'test@email.com';
+	const password = 'testpass';
+	const username = 'testuser';
+	const name = 'test name';
+
+	before(async () => {
+		signupResult = await signup(null, { email, password, username, name }, fakectx);
+		fakectx.req.headers.authorization = `Bearer ${signupResult.token}` // eslint-disable-line
+	});
+
+	after(async () => {
+		await User
+			.query()
+			.where({ id: signupResult.user.id })
+			.del();
+	});
+
+	it('should be able to confirm address link', async () => {
+		const network = 'kasuma';
+		const address = '5E9ap3x1fQ42vQ4kaD9UiSgDHZJ1WsPN64E1CMKBzAg3h7aX';
+		const signature = '0x660535f7dc76abff4de1ab61e194b2af9bfa88ba99de7c418e3a9ddbec2a463847031aad7e9f8634ccfdea4b8ae83723cc3997327b2760958ca04e3364cb3207';
```

Comment:
We should put the sign message in a const to make it more obvious.
```suggestion
		const sign_message = 'da194645-4daf-43b6-b023-6c6ce99ee709'
		const signature = '0x5c725c7f218b85e396cb58f16ecd57638fb6e97e0486efff752c519647e5b829b36900c6b1d602fc7b3407a7cdcf0ee78e4fa3db085fcb0bc06342b25d908d8c';
```

Reply:
done